### PR TITLE
darkmode

### PR DESCRIPTION
**Pull request by JTheilmann**
initial dark mode css based

### CODE REVIEW

**Comment by vrde** [comment]
Your changes look pretty good!

Should we make the main page dark as well, or at least not transparent?

image

PS: you can run `pnpm dev --mode production` to build the app in dev mode using production data.

**Comment by sirnicolaz** [comment]
looks good to me. @vrde if you can test it and it works, we can merge and deploy!

thanks @JTheilmann , nice job

**Review by sirnicolaz — APPROVED** [review]

File: www/src/Home.svelte [hunk 1]
```diff
@@ -175,38 +177,73 @@
...
-  .poem:after {
-    background: #f6f6f6;
-    box-shadow: 0 0 3px rgba(0, 0, 0, 0.2);
-    right: -3px;
-    top: 1px;
-    transform: rotate(1.4deg);
+  @media (prefers-color-scheme: dark) {
+      /* Inspired by https://codepen.io/dalper02/pen/VLeVjP */
+    .poem {
+      background: rgba(79, 79, 79, 0.177);
```
Comment (vrde) [inline www/src/Home.svelte:219]:
```suggestion
      background: #1e1e1e;
```

File: www/src/Home.svelte [hunk 2]
```diff
@@ -175,38 +177,73 @@
...
+    .poem:after {
+      content: "";
+      height: 98%;
+      position: absolute;
+      width: 100%;
+      z-index: -1;
+    }
+
+    .poem:before {
+      background: rgba(141, 141, 141, 0.177);
```
Comment (vrde) [inline www/src/Home.svelte:235]:
```suggestion
      background: rgba(21, 21, 21, 0.8);
```

File: www/src/Home.svelte [hunk 3]
```diff
@@ -175,38 +177,73 @@
...
+      left: -5px;
+      top: 4px;
+      transform: rotate(-2.5deg);
+    }
+
+    .poem:after {
+      background: rgba(73, 73, 73, 0.141);
```
Comment (vrde) [inline www/src/Home.svelte:242]:
```suggestion
      background: rgba(21, 21, 21, 0.8);
```

File: www/src/CurrentPoem.svelte [hunk 4]
```diff
@@ -157,7 +157,21 @@
...
-    background-color: #f3f3f3;
+  
+
+
+  @media (prefers-color-scheme: dark) {
+    .add-verse {
+      background: rgba(141, 141, 141, 0.177);
+    } 
+    .word {
+      color: #000;
```
Comment (vrde) [inline www/src/CurrentPoem.svelte:168]:
```suggestion
      color: #6c6c6c;
```

File: www/src/CurrentPoem.svelte [hunk 5]
```diff
@@ -157,7 +157,21 @@
...
   }
 
-  .add-verse {
-    background-color: #f3f3f3;
+  
+
+
+  @media (prefers-color-scheme: dark) {
+    .add-verse {
+      background: rgba(141, 141, 141, 0.177);
```
Comment (vrde) [inline www/src/CurrentPoem.svelte:165]:
```suggestion
      background: rgba(36, 36, 36, 0.8);
```

**Review by vrde — CHANGES_REQUESTED** [review]
I've added some suggestions to make the sheets of paper a bit darker and the current word more readable.

Would be nice to have the `textarea` with:

- border color `rgba(200, 200, 200, 0.5)`
- text color `rgb(110, 111, 128)`

This is how it would look:

image

One last thing, can you also fix the buttons here?

image

**Pull request merged**